components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -406,6 +446,20 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
     }
   }
 
+  private async _toPortForward() {
```

Comment:
Do we need to support port-forwarding via Cloud Shell Anymore?

I guess we can keep it for now and remove post-next.